api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder

```diff
@@ -16,11 +16,17 @@ json.average_rating app.average_rating
 json.add_on app.add_on?
 json.running_instances_count app.running_instances_count
 
-json.shared_entities do
-  json.array! app.shared_entities do |shared_entity|
-    json.extract! shared_entity, :shared_entity_nid, :shared_entity_name, :write, :read
+if app.app_shared_entities.any?
+  json.app_shared_entities do
+    json.array! app.app_shared_entities do |shared_entity|
+      json.shared_entity_nid shared_entity.shared_entity.nid
```

Comment:
because it's not taking the fields name by name, it is mapping 
shared_entity_nid to shared_entity.nid.